components/tools/OmeroJava/test/integration/chown/PermissionsTest.java

```diff
@@ -19,11 +19,14 @@
 
 package integration.chown;
 
+import static omero.rtypes.rstring;
```

Comment:
Ha ha, see https://github.com/openmicroscopy/openmicroscopy/commit/61cb92354986ca74eb3b9bc205362c7b7c21f511 then: @joshmoore must be fine with them. :+1: